analyzer/vulnerability-consumer/src/main/java/eu/fasten/analyzer/vulnerabilityconsumer/VulnerabilityConsumer.java

```diff
@@ -257,6 +274,7 @@ public static void writePatchDate(Vulnerability v) {
                 if (set)
                     v.setPatchDate(sdf.format(latestDate));
             } catch (Exception e) {
+                // TODO: Why an empty exception block?
```

Comment:
I can't explain it but I would suggest to remove it and implement a better error handling once we figure out which exceptions were silently catch.